book/pipelines.md

```diff
@@ -79,7 +79,7 @@ In effect, the command:
 And the pipeline:
 
 ```
-> ls | table
+> ls
```

Comment:
please don't change this one as it explains how `| table` is added for you automatically.

Reply:
very good catch, thanks @jntrnr :scream: 

should be fixed in f3fd7eba :+1:

Reply:
i've also added a note about `ls` and `ls | table` being *one and the same* in 17597f96, this is not true and caused more confusion in https://github.com/nushell/nushell/issues/9630 :thinking: